docs/v1/open_questions.md

```diff
@@ -0,0 +1,112 @@
+# Open Questions
+
+This document contains open questions regarding the
+[API design document](./design.md).
+
+## 1. Is the `ServiceInstanceBinding` resource necesssary?
+
+The `ServiceInstanceBinding` object is a byproduct of the successful execution
+of a `ServiceInstanceClaim`. Instead of writing a new `ServiceInstanceBinding`
+object, can the service catalog controller just add the binding information
+directly into the `ServiceInstanceClaim` on success?
```

Comment:
In order to reflect that the ServiceInstanceClaim has been satisfied, we should write the ServiceInstance back to the claim. We could write the other references back to the claim at the same if, if there were a reason to do so. I imagine the resources would link back to the claim for garbage collection if nothing else.

I'd be inclined to start without a binding resource, unless it serves a specific purpose.